Demo/src/main/java/com/paypal/android/ui/features/FeaturesView.kt

```diff
@@ -84,7 +95,13 @@ fun FeatureOptions(
 ) {
     Card(
         shape = CardDefaults.elevatedShape,
-        elevation = CardDefaults.elevatedCardElevation()
+        elevation = CardDefaults.elevatedCardElevation(),
+        modifier = Modifier
+            .padding(
+                start = UIConstants.paddingMedium,
+                end = UIConstants.paddingMedium,
+                bottom = UIConstants.paddingSmall
+            )
```

Comment:
More layout tweaks to address a visual bug that occurs when the `FeaturesView` has enough content to become scrollable.